CHANGELOG.md

```diff
@@ -3,15 +3,67 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.68.0
 
-- allow OpenTelemetry resource attributes to be configured under distributed_tracing config
+This release contains a mix of features and bugfixes.
+
+### Breaking Changes
+
+#### `entrypoint` annotation implies `document` scope
+
+The [entrypoint annotation's](https://www.openpolicyagent.org/docs/latest/policy-language/#entrypoint) scope requirement 
+has changed from `rule` to `document` ([https://github.com/open-policy-agent/opa/issues/6798](#6798)). 
+Furthermore, if no `scope` annotation is declared for a METADATA block preceding a rule, the presence of an `entrypoint` 
+annotation with a `true` value will assign the block a `document` scope, where the `rule` scope is otherwise the default.
 
 ### CLI
 
-- `opa exec`: This command never supported "pretty" formatting (`--format=pretty` or `-f pretty`), only `json`. Passing `pretty` is now invalid.
+- `opa exec`: This command never supported "pretty" formatting (`--format=pretty` or `-f pretty`), only `json`. Passing `pretty` is now invalid. ([#6923](https://github.com/open-policy-agent/opa/pull/6923)) authored by @srenatus
   Note that the flag is now unnecessary, but it's kept so existing calls like `opa exec -fjson ...` remain valid.
 
+### Topdown and Rego
+
+- ast: Change required scope of `entrypoint` from `rule` to `document` ([#6798](https://github.com/open-policy-agent/opa/issues/6798)) authored by @anderseknert
```

Comment:
Worth mentioning twice?

Reply:
Hm, yeah, maybe not 😄 .
I'll remove this and move the ticket link to the breaking change entry